packages/sdk-snippets/CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing
+
+For general contributing and commit guidelines for this project check out https://github.com/readmeio/.github/blob/main/.github/CONTRIBUTING.md.
+
+## Creating a new Target
+
+Much like in [HTTPSnippet](https://npm.im/httpsnippet) targets are programming languages. If you wish to create a new snippet generator for a new language follow these steps:
+
+1. Create a new directory in `src/targets/` and name it the language you're targeting. (eg. for PHP it'd be `php`, .NET `dotnet`).
+2. Create a `target.ts` file in this new directory with code similar to the following:
+
+```ts
+import type { Target } from '../targets';
+import { express } from './express/webhooks/client';
+
+export const node: Target = {
+  info: {
+    // This should be identical to the directory you created.
+    key: 'node',
+
+    // The proper naming of this language.
+    title: 'Node.js',
+
+    // The file extension that this language prefers. If it doesn't have a file
+    //  extension you can set this to an empty string.
+    extname: '.js',
+
+    // The default client this language will use. See below for instructions on
+    // creating a client.
+    default: 'native',
+
+    // This is how you execute the code that it generates.
+    cli: 'node %s',
+  },
+  services: {
```

Comment:
If the only difference across targets is the `clientById` child, is there an easier way to eliminate the boilerplate here while contributing?

Reply:
I'm cool to eliminate the `clientById` object prop entirely.